adapters/oidc/js/.browserslistrc

```diff
@@ -0,0 +1,6 @@
+IE 11
```

Comment:
This seemed like a good target for browser support for the statically server adapter. I'd like to have some kind of official acknowledgement of this though.